supertokens_python/recipe/session/api/implementation.py

```diff
@@ -40,6 +44,23 @@ class APIImplementation(APIInterface):
     async def refresh_post(
         self, api_options: APIOptions, user_context: Dict[str, Any]
     ) -> SessionContainer:
+
+        # If a request has multiple session cookies and 'older_cookie_domain' is
+        # unset, we can't identify the correct cookie for refreshing the session.
+        # Using the wrong cookie can cause an infinite refresh loop. To avoid this,
+        # we throw a 500 error asking the user to set 'older_cookie_domain'.
+        if (
+            has_multiple_cookies_for_token_type(api_options.request, "access")
+            or has_multiple_cookies_for_token_type(api_options.request, "refresh")
+        ) and api_options.config.older_cookie_domain is None:
+            raise Exception(
+                "The request contains multiple session cookies. This may happen if you've changed the 'cookie_domain' setting in your configuration. To clear tokens from the previous domain, set 'older_cookie_domain' in your config."
+            )
```

Comment:
Please move this into `clear_session_cookies_from_older_cookie_domain` to keep it consistent with node

Reply:
Fixed